code/datums/components/fullauto.dm

```diff
@@ -279,7 +279,7 @@
 		return FALSE
 	var/obj/item/bodypart/other_hand = shooter.has_hand_for_held_index(shooter.get_inactive_hand_index())
 	if(weapon_weight == WEAPON_HEAVY && (shooter.get_inactive_held_item() || !other_hand))
-		to_chat(shooter, span_warning("You need two hands to fire [src]!"))
+		balloon_alert(shooter, "need two hands!")
```

Comment:
needs perhaps?

Reply:
Actually after thinking a bit I think `use both hands!` is better wording, what do you think?

Reply:
Otherwise my reasoning for `need two hands` is that it is short for you "You need two hands"

Reply:
> Actually after thinking a bit I think `use both hands!` is better wording, what do you think?

i dig it